src/referencePackages/src/system.drawing.common/4.7.0/System.Drawing.Common.4.7.0.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Many `4.7.0` versions of packages have known vulnerabilities in them. Even though we don't include any of the vulnerable source code, they trigger CG alerts. In the instances where we've updated these projects in the past, you should use the checked-in updated version and update the reference from the dependent project. See here: https://github.com/dotnet/source-build-reference-packages/pulls?q=is%3Apr+4.7.0